CONTRIBUTING.rst

```diff
@@ -27,6 +27,17 @@ Quick Links for Contributing
 - To add a new service to OBS Studio please see the service submission guidelines:
   https://github.com/obsproject/obs-studio/wiki/Service-Submission-Guidelines
 
+General Guidelines
+------------------
+
+- The OBS Project uses English as a common language. Please ensure that any
+  submissions have at least machine-translated English descriptions and titles.
+
+- Templates for Pull Requests and Issues must be filled out. By not doing so
+  you are creating more work for the maintainers and yourself.
```

Comment:
I'd prefer something more like:

Templates for Pull Requests and Issues must be properly filled out. Failure to do so may result in your PR or Issue being closed. The templates request the bare minimum amount of information required for us to process them.

Reply:
Sure, let's go with that.